content/applications/sales/rental.rst

```diff
@@ -0,0 +1,93 @@
+======
+Rental
+======
+
+**Odoo Rental** is a comprehensive solution to manage your rentals.
+
+From a single view, you can send out quotations, confirm orders, schedule rentals, register when
+products are picked up and returned, and invoice your customers.
+
+.. seealso::
+   - `Odoo Rental: product page <https://www.odoo.com/app/rental>`_
+
+Configuring rental pricing
+==========================
+
+Go to :menuselection:`Rental --> Products`, select or create a product, and click on the
+product's *Rental tab*. Under *Rental Pricing*, click on *Add a price*. Then you choose a *Unit*
+of time (hours, days, weeks, or months), a *Duration*, and a *Price*. You can add as many price
+lines as necessary, usually to give out discounts for longer rental durations.
+
+.. image:: rental/rental-pricing-example.png
+   :align: center
+   :alt: Example of rental pricing configuration in Odoo Rental
+
+.. tip::
+   Under *Reservations*, you can add fines for any *Extra Hour* or *Extra Day*. You can also set a
+   *Security Time*, expressed in hours, to make the product temporarily unavailable between two
+   rental orders.
+
+.. note::
+   If you want to rent a product created outside of the Rental app, do not forget to tick *Can be
+   Rented* under the product's name. By default, this option is ticked when you create a product
+   directly from the Rental app.
+
+Computing rental pricing
+------------------------
+
+Odoo always uses two rules to compute the price of a product when you create a rental order:
+
+#. Only one price line is used.
+#. The cheapest line is selected.
+
+Consider the following rental pricing configuration for a product:
+
+- 1 day: $100
+- 3 days: $250
+- 1 week: $500
+
+A customer wants to rent this product for 8 days. After an order is created, Odoo selects the
+second line as this is the cheapest option. The customer has to pay 3 times '3 days' to
+cover the rental's 8 days, for a total of $750.
+
+Requesting a customer signature
+===============================
+
+You can ask your customers to sign a rental agreement outlining the arrangement between you and
+your customers before they pick up products to make sure your products are returned on time and
+in their original condition. To do so, go to :menuselection:`Rental --> Configuration
+--> Settings`, activate *Digital Documents* and *Save*.
+
+.. image:: rental/digital-documents-settings.png
+   :align: center
+   :alt: Digital Documents settings in Odoo Rental
+
+.. note::
+   This feature requires the *Sign* app. If necessary, Odoo installs it after activating *Digital
+   Documents*.
```

Comment:
I'm wondering if to mention that it can increase their subscription. There is a warning message in odoo anyway. Up to you :)
(and if you add it, maybe make it a warning rather than a note?)

+ add doc reference

```suggestion
.. note::
   This feature requires the :doc:`Sign <../finance/sign>` app. If necessary, Odoo installs it after
   activating *Digital Documents*.
```

Reply:
I have never seen that warning message in Odoo. But that's maybe because I work on the runbot or usually install ALL apps on my own DBs. Anyway, as long as there is a warning in Odoo I guess it's not really necessary to add here.